src/Symfony/Component/Messenger/EventListener/StopWorkerOnSigtermSignalListener.php

```diff
@@ -11,14 +11,15 @@
 
 namespace Symfony\Component\Messenger\EventListener;
 
-trigger_deprecation('symfony/messenger', '6.3', '"%s" is deprecated, use "%s" instead.', StopWorkerOnSigtermSignalListener::class, StopWorkerOnSignalsListener::class);
+trigger_deprecation('symfony/messenger', '6.3', '"%s" is deprecated, use the "%s" instead.', StopWorkerOnSigtermSignalListener::class, SignalableCommandInterface::class);
```

Comment:
this should be moved after the corresponding `use` to work

Reply:
Fixed, thanks :man_facepalming: